lib/protocol/packets/RowDataPacket.js

```diff
@@ -13,6 +13,22 @@ Object.defineProperty(RowDataPacket.prototype, 'parse', {
   value        : parse
 });
 
+Object.defineProperty(RowDataPacket.prototype, 'isArray', {
```

Comment:
Can these be at least underscore-prefixed? Whatever we add here will shadow any column someone may be trying to query, so need to be careful adding anything new.